data/campaigns/Heir_To_The_Throne/scenarios/07_Crossroads.cfg

```diff
@@ -375,30 +375,69 @@
 
         # Only the number of ambushers differs by difficulty here.
 
-#ifdef EASY
-        {ORC_AMBUSH_AREA 2 5-35 1-7 5}
-        {ORC_AMBUSH_AREA 2 9-24 8-15 5}
-        {ORC_AMBUSH_AREA 3 1-17 17-24 9}
-        {ORC_AMBUSH_AREA 3 18-25 16-23 5}
-        {ORC_AMBUSH_AREA 3 26-40 16-23 5}
-#endif
-#ifdef NORMAL
-        {ORC_AMBUSH_AREA 2 5-35 1-7 6}
-        {ORC_AMBUSH_AREA 2 9-24 8-15 6}
-        {ORC_AMBUSH_AREA 3 1-17 17-24 10}
-        {ORC_AMBUSH_AREA 3 18-25 16-23 6}
-        {ORC_AMBUSH_AREA 3 26-40 16-23 6}
-#endif
-#ifdef HARD
-        {ORC_AMBUSH_AREA 2 5-35 1-7 7}
-        {ORC_AMBUSH_AREA 2 9-24 8-15 7}
-        {ORC_AMBUSH_AREA 3 1-17 17-24 11}
-        {ORC_AMBUSH_AREA 3 18-25 16-23 7}
-        {ORC_AMBUSH_AREA 3 26-40 16-23 7}
-#endif
+        # Adjust for leaders killed in S01 ('The Elves Besieged')
+        # Reduce 25% for each. Note: implicit floor() due to loop
+        # GL-2016Aug
+        #
+        #             3  2  1  0 # killed
+        #
+        #      5 -->  1  2  3  5
+        #      6 -->  1  3  4  6
+        #      7 -->  1  3  5  7
+        #
+        #      9 -->  2  4  6  9
+        #     10 -->  2  5  7 10
+        #     11 -->  2  5  8 11
+        #
+        # Net result
+        #
+        #   Easy -->  6 12 18 29
+        # Normal -->  6 17 23 34
+        #   Hard -->  6 17 28 39
+
+
+        [set_variable]
+            name=small_number
+            value=4
+        [/set_variable]
+        [set_variable]
+            name=small_number
+            sub="0$besieged_enemies_killed"
```

Comment:
I also wonder whether it'd be easier if you put those wholeset_variable call in one forumla expression, like in 
```{VARIABLE small_number "$((4 - $besieged_enemies_killed?0|) * {ON_DIFFICULTY 5 6 7} / 4)"}```